### ISSUE TITLE
Decision Jam tile image was broken

### ISSUE THREAD

**Issue by tdbeattie** [openpracticelibrary/openpracticelibrary#881]
I noticed the tile image wasn't working for the recently added Decision Jam practice. I've temporarily taken it out (so stock image now shows) and adding this issue to replace that with proper image via new PR.

<img width="837" alt="Screenshot 2020-01-29 at 16 31 28" src="https://user-images.githubusercontent.com/6207402/73376340-34c41180-42b5-11ea-9f19-4614cddc26e2.png">

**Comment by heyjerrybecker** [comment]
Cool - thanks for doing that @tdbeattie - Now that it's up and working, I tried going back in and adding the tile image I originally wanted. We'll see if it works!

**Comment by tdbeattie** [comment]
@heyjerrybecker - I found a workaround for this as the same problem occurred in another PR I was looking at.

I think the rendering problem is with higher res images. When I reduced this down to 476 × 326, it seems to work. I've applied this to you Decision Jam with PR https://github.com/openpracticelibrary/openpracticelibrary/pull/893 with deploy preview https://deploy-preview-893--openpracticelibrary.netlify.com/ - take a look and, if it's what you were wanting, merge away!
